test/wdio.sauce.js

```diff
@@ -1,4 +1,4 @@
-var config = require('./wdio.conf').config;
+var config = require('./wdio.quick').config;
```

Comment:
Yeah, we don't want the full data tests run on every Circle build. We do, however, want to run them at some point soon (probably locally?) and then use them to work out & fix discrepancies. @hollyallen, when/where do you think we should be doing that in relation to our current schedule?